packages/beta/src/__tests__/api/alertsApi.int.spec.ts

```diff
@@ -290,9 +269,9 @@ describe('alerts api', () => {
           order: 'desc',
         },
       })
-      .autoPagingToArray({ limit: 1001 });
+      .autoPagingToArray({ limit: 50 });
 
-    expect((await alerts).length).toBeGreaterThan(1000);
+    expect((await alerts).length).toBe(50);
```

Comment:
I don't understand how this tests pagination. Aren't we just fetching 50 alerts without pagination here?
Also `autoPagingToArray({ limit: 50 });` will not provide a `limit` query param to the API as this is logic for the SDK. If you want to send a `limit` query param to the API, then it should be here:
```js
const alerts = client.alerts.list({..., limit: 25})`
```